src/index.html

```diff
@@ -9,6 +9,51 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+          class="logo-img"
+          src="./images/logo.png"
+          alt="asd">
```

Comment:
Fix it in all cases
![image](https://github.com/mate-academy/layout_moyo-header/assets/91411692/87d52286-4e42-4858-b4e3-67ce82120600)
```suggestion
        <img
          class="logo-img"
          src="./images/logo.png"
          alt="asd"
        >
```